workspaces/mend/plugins/mend-backend/src/plugin.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  coreServices,
+  createBackendPlugin,
+} from '@backstage/backend-plugin-api';
+import { createRouter } from './service/router';
+
+/**
+ * mendPlugin backend plugin
+ *
+ * @public
+ */
+export const mendPlugin = createBackendPlugin({
+  pluginId: 'mend',
+  register(env) {
+    env.registerInit({
+      deps: {
+        auth: coreServices.auth,
+        config: coreServices.rootConfig,
+        discovery: coreServices.discovery,
+        httpAuth: coreServices.httpAuth,
+        httpRouter: coreServices.httpRouter,
+        logger: coreServices.logger,
+        permissions: coreServices.permissions,
+      },
+      async init({
+        auth,
+        config,
+        discovery,
+        httpAuth,
+        httpRouter,
+        logger,
+        permissions,
+      }) {
+        httpRouter.use(
+          await createRouter({
+            auth,
+            config,
+            discovery,
+            httpAuth,
+            logger,
+            permissions,
+          }),
+        );
+        httpRouter.addAuthPolicy({
+          path: '/health',
+          allow: 'unauthenticated',
+        });
```

Comment:
I don't see this in your `router.ts`, you should remove this then.

Reply:
https://github.com/backstage/community-plugins/blob/c2f9428c853044886c898037e5a0aa528ce83ed8/workspaces/mend/plugins/mend-backend/src/service/router.ts#L273

Reply:
Thanks, sorry I missed that :(